app/helpers/collection_helper.rb

```diff
@@ -1,4 +1,9 @@
 module CollectionHelper
+  # strip leading 'a' from collection catkeys
+  def self.strip_leading_a(collection_id)
+    collection_id.sub(/^a(\d+)$/, '\1')
```

Comment:
Not functionally different but worth considering a string-based implementation?

```suggestion
    collection_id.delete_prefix('a')
```

Reply:
I used the pattern that was previously used but I think this probably makes more sense.

Reply:
We've been using `sub(/^a(\d+)$/, '\1')` to avoid accidentally stripping an `a` from catkey that wasn't migrated over from symphony (where a leading `a` might happen.. I'm not sure we have any like this yet, but...)

Reply:
Okay. Changed it back.